symengine/functions.cpp

```diff
@@ -165,8 +165,7 @@ bool could_extract_minus(const Basic &arg)
     if (is_a_Number(arg)) {
         if (down_cast<const Number &>(arg).is_negative()) {
             return true;
-            // TODO: see #915
-        } else if (is_a<Complex>(arg)) {
+        } else if (is_a_Complex(arg)) {
             const Complex &c = down_cast<const Complex &>(arg);
```

Comment:
`is_a_Complex()` function is fine as it is. What I mean is you can't down cast `arg` since it might not be a `Complex` object. (It is an object of any of the subclasses of ComplexBase)